src/main/scala/cromwell/engine/package.scala

```diff
@@ -107,4 +108,22 @@ package object engine {
     type ExecutionStatus = Value
     val NotStarted, Starting, Running, Failed, Done, Aborted, Aborting = Value
   }
+
+  /*
+   * Type and implicit conversion methods for ExecutionIndex
+   */
+  object ExecutionIndex {
```

Comment:
Oh whoops, this is not quite what I read it to be the first time; my earlier comments might not make sense.  This might work better as a pair of implicit classes to do `Option[Int]` <--> `Int` by "enhancing" those types with no-arg versions of methods like these.